client/src/components/geneExpression/index.js

```diff
@@ -17,11 +17,14 @@ class GeneExpression extends React.Component {
     const { genesets } = this.props;
 
     for (const [name, geneset] of genesets) {
+      const isDiffExp = geneset.genesetName.includes("DiffExp Set");
```

Comment:
Is it the case here that `DiffExp Set` is a magic word, and does the for loop have to know about it? Could it be extracted to globals as a constant?

Reply:
Yes to magic word, the loops uses this to know if the geneset is a diffexp geneset. I'll extract it to a global const

Reply:
On second thought, this might not be the best idea.  If a user renames this geneset they'll lose diffexp functionality which would be unexpected functionality.

I'll most likely shift over to a geneset description instead, any suggestions for what the insert into the geneset description? 
cc: @bkmartinjr , @colinmegill

Reply:
Spoke with colin offline, we'll be moving forward with description